test_scripts/Polices/build_options/ATF_P_TC_SDL_Build_EXTENDED_POLICY_HTTP.lua

```diff
@@ -54,33 +54,37 @@ end
 function Test:TestStep_Trigger_PTU_Check_HTTP_flow()
   local corId = self.mobileSession2:SendRPC("RegisterAppInterface", config.application2.registerAppInterfaceParams)
   EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application2.registerAppInterfaceParams.appName }})
-   :Do(function()
-     EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
-      :Do(function()
-          local CorIdSystemRequest = self.mobileSession:SendRPC ("SystemRequest",
-            { requestType = "HTTP", fileName = "PTU" }, filePTU)
-          EXPECT_RESPONSE(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
-          
-        end)
-      self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
-      self.mobileSession2:ExpectNotification("OnPermissionsChange")
-   end)
-    EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate")
-     :ValidIf(function(exp,data)
-       if exp.occurences == 1 and data.params.status == "UPDATE_NEEDED" then
-         return true
-       elseif exp.occurences == 2 and data.params.status == "UPDATING" then
-         return true
-       elseif exp.occurences == 3 and data.params.status == "UP_TO_DATE" then
-         return true
-        end
-        return false
+
+  EXPECT_NOTIFICATION("OnSystemRequest")
```

Comment:
@istoimenova, Since there are 2 apps registered and they are in NONE HMI level OnSystemRequest can be sent through any of them